### PR TITLE
Added Venezuela Playbook

### LINKED ISSUE #258: Playbook: Venezuela (VE)
### Country

Venezuela (VE)

### eAIP

_No response_

### Steps to add a new playbook

You can check [playbooks](https://github.com/drph4nt0m/avbot-charts/tree/main/playbooks) for examples on chart playbooks and [axios engine](https://github.com/drph4nt0m/avbot-charts/blob/main/app/engines/axios.js) on how the features in the playbook are used by the axios scraping engine.

Tasks summary:
-[ ] Fork & clone AvBot Charts
-[ ] Create playbook for the country in playbooks
-[ ] Add tests for the country in tests/getChart.test.js (tests should be in alphabetical order of the country code)
-[ ] Push changes and submit a pull request
-[ ] If you need any help, reach out to us on our [Discord server](http://link.avbot.in/support).

Are you ready to work on this issue? Let us know, and we will assign it to you.

Happy Coding!

### PR DESCRIPTION
**Pull request by Harsh-Singh-Rajput**
# Description

Added playbook for Venezuela and also added tests.

Fixes #258 

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [X] Test A
- [X] Test B

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation/readme
- [X] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by akgupta0777 — APPROVED** [review]
Thanks 👍

**Comment by drph4nt0m** [comment]
@Harsh-Singh-Rajput please run `npm run prettier` and repush the changes.

**Review by dmarkowicz — CHANGES_REQUESTED** [review]

**Comment by Harsh-Singh-Rajput** [comment]
> @Harsh-Singh-Rajput please run `npm run prettier` and repush the changes.

ok will try

File: playbooks/VE.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "country": {
+      "iso": "VE",
+      "name": "Venezuela"
+    },
+    "source": "Servicios de venezuela",
+    "scraper": {
+      "engine": "axios",
+      "features": {
+        "baseUrl": "http://www.inac.gob.ve/eaip/2018-07-19/html",
```
[the remark this answers is not in the document]
Comment (akgupta0777) [reply playbooks/VE.json:10]: image

Actually, he used the currently effective AIP.At present the older eAIP is being used I think.

[the remark this answers is not in the document]
Comment (dmarkowicz) [reply playbooks/VE.json:10]: I think the website is out of date, but of course you may be right.

**Review by dmarkowicz — DISMISSED** [review]

File: playbooks/VE.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Harsh-Singh-Rajput) [reply playbooks/VE.json]: i changed the url to the new one @dmarkowicz

[the remark this answers is not in the document]
Comment (drph4nt0m) [reply playbooks/VE.json]: Yeah, i also feel the website is outdated. Thanks @dmarkowicz for pointing that out.

**Review by drph4nt0m — APPROVED** [review]

**Comment by drph4nt0m** [comment]
@Harsh-Singh-Rajput thanks for your contribution.

**Comment by Harsh-Singh-Rajput** [comment]
> @Harsh-Singh-Rajput thanks for your contribution.

I'm Happy to contribute learnt something new 🙂

**Pull request merged**